docs/pages/eas-update/introduction.mdx

```diff
@@ -18,7 +19,38 @@ All apps that include the `expo-updates` library have the ability to receive upd
 - To start using EAS Update, continue to the [Getting Started](/eas-update/getting-started) guide.
 - For a complete tutorial of using EAS Update with other EAS services, refer to the [EAS Tutorial](/tutorial/eas/introduction/).
 
-> **info** For a look at some of our favorite features of EAS Update, check out the [6 reasons to use EAS Update blog post](https://expo.dev/blog/6-reasons-to-use-eas-update).
+## JS API for update management
+
+<ContentSpotlight
+  alt="Richer JS Api."
+  src="/static/images/eas-update/frontpage/critical-update.png"
+/>
+
+An improved [JavaScript API](https://docs.expo.dev/versions/unversioned/sdk/updates/) enhances the app update process, including a React hook called `useUpdates()`. This hook provides developers with detailed information about the currently running app bundle and any new updates that are available or have been downloaded. It is linked to a native state machine, ensuring it reflects the most current state of the system at all times. The `useUpdates()` hook can be utilized from any component within the application and keeps track of when the device last queried the update server.
```

Comment:
```suggestion
The updates [JavaScript API](/versions/unversioned/sdk/updates/) includes a React hook called `useUpdates()`. This hook provides detailed information about the currently running update and any new updates that are available or have been downloaded. In addition, you can view any errors that were encountered during the update process to help you debug any issues while the app is attempting to update.
```

- Replaced "app bundle" with "update", since we don't call them app bundles generally
- Removed the part about how `useUpdates()` works (it uses a state machine). I would expect this as a developer and not sure how this implementation detail is relevant.
- The last sentence that the `useUpdates()` hook can be utilized in any component within an application seems like a rephrase of the first and second sentences. To make this more concise, I removed that one too.